### DISCUSSION TITLE
How to change Trend flexibility?

### DISCUSSION THREAD

**Discussion by zanynik** [ourownstory/neural_prophet#283] [Q&A - get help using NeuralProphet]
Hi, 
How to void this kind of forecast ? For this I had set n_changepoints = 30.

Should I try tend_smoothness ? Where to find how to use this arguement and what it does ? 

![image](https://user-images.githubusercontent.com/10960801/108893115-ff513b00-7610-11eb-923f-2e042418e8bf.png)

**Comment by ourownstory** [comment]
Hi @zanynik Do you mind sharing with us what data  you are using and your model setup?
